qemu/tests/remote_image_multiwrite.py

```diff
@@ -0,0 +1,45 @@
+import re
+import logging
+
+from virttest import error_context
+from virttest import utils_disk
+from virttest import utils_misc
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    1) Start guest with both data disk and system disk.
+    2) Format data disk(only ext4 is tested by now)
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    session = None
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login(timeout=int(params.get("login_timeout", 240)))
+
+    stg_tag = params["images"].split()[-1]
+    stg_params = params.object_params(stg_tag)
+    stg_fstype = stg_params["disk_format"]
+    stg_size = stg_params["image_size"]
+    stg_extra_params = stg_params.get("blk_extra_params", "")
+    match = re.search(r"(serial|wwn)=(\w+)", stg_extra_params, re.M)
+
+    try:
+        drive_id = match.group(2)
+        drive_path = utils_misc.get_linux_drive_path(session, drive_id)
+        did = drive_path[5:]
+        logging.info("Format %s(size=%s) with %s type." % (did, stg_size,
+                                                           stg_fstype))
+        mnts = utils_disk.configure_empty_linux_disk(session, did, stg_size,
```

Comment:
did can be got by \"did = utils_disk.get_linux_disks(session).keys()\"
if then line 29 to line 35 can be removed, and line 20 to line 22 in cfg can be removed.

Reply:
Yes, this is another way to get device id, but the disadvantage is we cannot explicitly point out the first disk of 'utils_disk.get_linux_disks(session).keys()' is exactly the data disk to be formatted from code or conf, for a new contributor, he/she might have to check the implementation of get_linux_disks to find out what it does; while the current way is quite straightforward; Either way, IMO, would be OK.